src/main/java/lanat/helpRepresentation/ArgumentRepr.java

```diff
@@ -38,7 +53,7 @@ private ArgumentRepr() {}
 		return outText.toString();
 	}
 
-	public static @Nullable String getDescription(@NotNull Argument<?, ?> arg) {
+	public static @Nullable String getDescription(final @NotNull Argument<?, ?> arg) {
```

Comment:
Is the final modifier really needed?
```suggestion
	public static @Nullable String getDescription(@NotNull Argument<?, ?> arg) {
```